frontend/package-lock.json

```diff
@@ -40,6 +40,8 @@
         "prop-types": "^15.8.1",
         "query-string": "^7.1.1",
         "react": "^18.0.0",
+        "react-datepicker": "^4.8.0",
```

Comment:
react-datepicker is not used anymore. It should be removed from package-lock.json